apps/dsp-app/src/app/user/user-form/user-form-constants.ts

```diff
@@ -0,0 +1,21 @@
+import { ValidationMessages } from './user-form-model';
+
+export const validationMessages: ValidationMessages = {
```

Comment:
look at reusable-project-form, it looks much better for field validation! This code seems like the old way of doiing things